styleSheet.css

```diff
@@ -302,32 +314,36 @@ ul {
     margin-left: 82%;
   }
 
+  #bodyResume {
+    margin-top: 0;
+  }
+
   #home {
-    height: 90%;
+    height: calc(65vh + 80vw);
   }
 
-  .home__right{
+  .home__right {
     display: none;
   }
 
-  .home__left{
+  .home__left {
     width: 100%;
   }
 
-  .home__profilePicture{
+  .home__profilePicture {
     margin: 5% 0 0 35%;
     width: 30%;
   }
 
-  .home__profileName{
+  .home__profileName {
```

Comment:
if you have a two words element you can use `-` . Example `.home__profile-name`

Reply:
Right, but I have been using that naming scheme (`.home__profileName`), Should I rename all of them?